source/DasBlog.Web.UI/Properties/launchSettings.json

```diff
@@ -21,7 +21,7 @@
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development"
       },
-      "applicationUrl": "http://localhost:50432/"
+      "applicationUrl": "https://localhost:5001;http://localhost:5000"
```

Comment:
We are also switching to Kestrel for local dev which is more reliable than IIS Express